src/Core/Managed/Shared/Extensibility/Implementation/Property.cs

```diff
@@ -141,8 +141,15 @@ public static void SanitizeMeasurements(this IDictionary<string, double> diction
 
                     string sanitizedKey = SanitizeKey(entry.Key, dictionary);
 
+                    // Disallow Nan and Infinity since Breeze does not accept it
+                    double sanitizeValue = entry.Value;
```

Comment:
We have it also in Metric and in Sampling. I updated the first one and I do mot think we need to update the second.